src/component/App.js

```diff
@@ -1,12 +1,46 @@
 import './App.css';
-import DataTable from './DataTable'
+
+import firebase from 'firebase/app';
+import 'firebase/firestore';
+import 'firebase/auth';
+
+import { useCollectionData } from 'react-firebase-hooks/firestore';
```

Comment:
The hook useCollectionData is for firestore too - can you research which hooks to use with realtime database?